.changelogs/8502.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Updated HyperFormula to 1.1.0",
```

Comment:
More like `^1.x.x`. And a short explanation why this is a breaking change. Maybe @kirszenbaum can help how to fit it in a single sentence? We will probably go into details in a blog post, but should be more informative at a glance. i.e. if someone is not using formulas will know that "Breaking change" is not for them

Reply:
```suggestion
  "title": "Updated the optional HyperFormula dependency from 0.6.2 to ^1.0.0, which introduces breaking changes for the `formula` plugin users.",
```